src/display.cpp

```diff
@@ -97,14 +103,14 @@ namespace Display {
     }
 
     DString &DString::resetColor() {
-        return *this << "\x1b[0m";
+        return *this << CSI + "0m";
     }
 
     DString &DString::saveCursorPosition() {
         // While the first is the official ANSI code for saving the cursor position,
         // the latter works in most unix terminals, but not MacOS.
 #ifdef __WIN32__
-        *this += "\x1b[s";
+        *this += CSI + "s";
 #elif __linux__
         *this += "\x1b"
```

Comment:
Use CSI

Reply:
Interesting - I think I pushed an old version, but cannot find the one where I forgot the CSI...